extension/memorylimiterextension/memorylimiter.go

```diff
@@ -34,7 +34,6 @@ func (ml *memoryLimiterExtension) Shutdown(ctx context.Context) error {
 	return ml.memLimiter.Shutdown(ctx)
 }
 
-// MustRefuse returns if the caller should deny because memory has reached it's configured limits
```

Comment:
Why removed?

Reply:
prob when i made the interface and moved it there and also moved the doc